docs/docs/reference/NoirJS/noir_js/index.md

```diff
@@ -26,7 +26,7 @@
 | :------ | :------ |
 | [and](functions/and.md) | Performs a bitwise AND operation between `lhs` and `rhs` |
 | [blake2s256](functions/blake2s256.md) | Calculates the Blake2s256 hash of the input bytes |
-| [ecdsa\_secp256k1\_verify](functions/ecdsa_secp256k1_verify.md) | Verifies a ECDSA signature over the secp256k1 curve. |
+| [ecdsa\_secp256k1\_verify](functions/ecdsa_secp256k1_verify.md) | Calculates the Blake2s256 hash of the input bytes and represents these as a single field element. |
```

Comment:
This seems wrong.

Reply:
```suggestion
| [ecdsa_secp256k1_verify](functions/ecdsa_secp256k1_verify.md) | Verifies a ECDSA signature over the secp256k1 curve. |
```

Reply:
I don't even know why this file was committed. It is autogenerated and will be autogenerated at build time. Could be worth checking if the typedoc decorator itself is wrong, I'll have a look

Reply:
Fixed as explained above 👍